src/pages/CreateEvent/ChallengeDetails.scss

```diff
@@ -13,10 +13,10 @@
       margin-bottom: 20px;
   
       .step {
```

Comment:
I believe we are moving towards tailwind and shadcn but if you do use SCSS, you might want to use BEM as well. That way we prevent additional nesting that isn't needed.